packages/components/upload/src/ajax.ts

```diff
@@ -70,10 +70,11 @@ export const ajaxUpload: UploadRequestHandler = (option) => {
   const formData = new FormData()
   if (option.data) {
     for (const [key, value] of Object.entries(option.data)) {
-      if (isArray(value) && value.length) formData.append(key, ...value)
+      if (isArray(value) && value.length) formData.append(key, value.join(','))
```

Comment:
It seems that the stitching of multiple values ​​into one value also changes the original logic.